kbase-extension/kbase_templates/page.html

```diff
@@ -82,7 +81,7 @@
             jquery: 'components/jquery/jquery.min',
             bootstrap: '{{ static_url("ext_components/bootstrap/dist/js/bootstrap.min") }}',
             bootstraptour: 'components/bootstrap-tour/build/js/bootstrap-tour.min',
-            'jquery-ui': 'components/jquery-ui/jquery-ui.min',
+            'jquery-ui': 'ext_components/jquery-ui/dist/jquery-ui.min',
```

Comment:
has jQ UI been added to the dependencies somewhere or is it in a directory that gets served up by jupyter? (Sorry, I have forgotten where the `ext_components` directory gets its contents from...)

Reply:
Uh, both.
It was apparently added to our dependencies a long time back (that's the `ext_components` path), but we were using the one served up by Jupyter (in `components`) but that's been either moved somewhere else or removed all together.

I just moved that to use our install.

Reply:
Glad that we are so committed to ensuring that jQ UI is on the narrative image!